pkg/resourcemanager/psql/vnetrule/client.go

```diff
@@ -83,10 +83,15 @@ func (c *PostgreSQLVNetRuleClient) CreateOrUpdatePostgreSQLVNetRule(
 	VNetRG string,
 	VNetName string,
 	SubnetName string,
+	subscription string,
 	IgnoreServiceEndpoint bool) (vnr psql.VirtualNetworkRule, err error) {
 
 	VNetRulesClient := GetPostgreSQLVNetRulesClient(c.creds)
```

Comment:
Should `GetPostgreSQLVNetRulesClient()` also accept the VNet SubscriptionID as a parameter?

It uses `creds.SubscriptionID()` but it's not clear to me which of the two subscriptions it needs.

Reply:
No, it's right that it uses the subscription id from the creds. It will still make the changes in ASO's subscription, the firewall rule will just refer to the vnet in the other subscription.